src/coord/tests/testdata/t

```diff
@@ -30,13 +30,13 @@ error: unknown catalog item 'bar'
 resolve
 SELECT 1 FROM materialize.public.foo
 ----
-SELECT 1 FROM [u1: materialize.public.foo]
+SELECT 1 FROM [u1 AS materialize.public.foo]
 
 resolve
 WITH foo AS (SELECT 1)
 SELECT 1 FROM foo
 ----
-WITH foo AS (SELECT 1) SELECT 1 FROM [l0: foo]
+WITH foo AS (SELECT 1) SELECT 1 FROM [l0 AS foo]
```

Comment:
Views containing ```WITH``` are unfortunately not persisted in a way that is readable on startup:

This view definition:

```
create view v10 as with a1 as (select * from t1) select * from a1;
```
is persisted as:
```
CREATE VIEW "materialize"."public"."v10" AS WITH "a1" AS (SELECT * FROM [u11 AS "materialize"."public"."t1"]) SELECT * FROM [l0 AS "a1"]
```
which fails when read on startup:
```
materialized: corrupt catalog: failed to deserialize item u18 (materialize.public.v10): table name in square brackets must be fully qualified
```